apps/molgenis-components/src/queryEmx2/queryEMX2.test.ts

```diff
@@ -94,7 +94,7 @@ Biobanks(filter: { _and: [ { name: { like: "UMC" } } ] }) {
       .getQuery();
 
     expect(query).toEqual(`{
-Biobanks(filter: { _or: [ { country: { name: { like: "DE" } } }, { collections: { name: { like: "covid" } } } ] }) {
+Biobanks(filter: { _and: [ { _or: [ { country: { name: { like: "DE" } } } ] }, { _or: [ { collections: { name: { like: "covid" } } } ] } ] }) {
```

Comment:
the original filter seems to be correct?

the change would be equivalent to [{ country: { name: { like: "DE" } } } ,{ collections: { name: { like: "covid" } } } ] as default is _and and _or arrays with single values have no added value.

Reply:
or is this just an example of two 'or' clauses that should both be true?